requirements-optional.txt

```diff
@@ -16,4 +16,5 @@ pycodestyle==2.5.0
 mypy==0.761
 pydocstyle==5.0.1
 flake8==3.7.9
-pylint==2.4.4
\ No newline at end of file
+pylint==2.4.4
+plotly==3.10.0
```

Comment:
This is an out-of-date plotly version, please bump to latest

Reply:
Updated